imagersite/imagerprofile/templates/album.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
+{% block main %}
+  <main>
+    <div class='album'>
+      <img src="{{ MEDIA_ROOT }}{{ album.cover.image.url }}">
+      <ul>
+        {% for photo in album.photos.all %}
+          <li>
+            <a href="{% url 'photo_page' photo_id=photo.id %}">
+              {% include "include_photo.html" %}
+            </a>
+          </li>
+        {% endfor %}
+      </ul>
+    </div>
+  </main>
+{% endblock %}
+
+{% block nav %}
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'edit_page' model='album' model_id=album.id %}">Edit</a></li>
+  {% endif %}
+{% endblock %}
```

Comment:
Does this template belong here in the Profile app?  It's a view for a model defined in the photo app, shouldn't it be located there instead?